src/features/search/index.tsx

```diff
@@ -74,36 +68,35 @@ export const Search: FC = () => {
     const search: Search = useCallback((query: string) => dispatch(updateTicker(query)), [query, dispatch]);
     const errorQuote = ''
 
+    const stockList = useSelector((state: AppState) => state.stockList)
+
     useEffect(() => {
         if(errorQuote.length > 0) {
-            setStockList([{name: errorQuote, symbol:'⊗'}])
+            dispatch(updateStockList([{name: errorQuote, symbol:'⊗'}]))
         }
     }, [errorQuote])
 
     useEffect(() => {
         toggleIsOpen(stockList.length !== 0)
     },[ stockList.length])
 
-    useEffect(() => {
-        socket.on('search', setStockList)
-        return () => void socket.off('search', setStockList);
-    }, []);
-
     useEffect(() => {
         if(query === '') {
-            setStockList([]);
+            dispatch(updateStockList([]))
             return;
         }
 
         socket.emit('search', query);
     }, [query]);
 
+    console.log(stockList)
```

Comment:
Remove `console.log`s, please

Reply:
sorry didn't spot this guy, changed and will be pushed with additional requested changes.